dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -428,16 +409,6 @@ public void setCheckerWorkflow(BioWorkflow checkerWorkflow) {
         this.checkerWorkflow = checkerWorkflow;
     }
 
-    /**
-     * @deprecated since 1.14.0. Use getAuthors instead.
-     */
-    @JsonProperty
-    @Deprecated(since = "1.14.0")
-    public String getAuthor() {
```

Comment:
Do you have a UI PR to go along with this? Removing this will break the UI since there's still some usage of author/email there (should double check). See description of https://github.com/dockstore/dockstore-ui2/pull/1755